spec/controllers/ops_settings_spec.rb

```diff
@@ -137,7 +137,6 @@
       end
 
       it "#does not allow duplicate names when adding" do
-        miq_server = EvmSpecHelper.local_miq_server
```

Comment:
not sure about this one, we do use `EvmSpecHelper.local_miq_server` for side effects in 30+ places

(but agreed with the `miq_server = ` part :))